src/buildercore/trop.py

```diff
@@ -912,16 +911,15 @@ def entry(hostname, i):
                     GetAtt(ELB_TITLE, "DNSName")
                 )
             )
-        else:
```

Comment:
no `else` can contain a `return` essentially? No strong opinion, but if it pushes for simpler functions it can be a good thing

Reply:
I think that's the jist of it and I agree - it's minutely simpler. 

It also gives an official stamp to one of my own pet peeves ;)